### PR TITLE
Reword the help text in the registration dialog (JSC#SLE-24216)

### PR DESCRIPTION
**Pull request by cwickert**
## Problem

I was asked to write a decent help text for the registration dialog. I think the current text is okay, I just reworked it a bit and fixed some errors.

For the background see in https://jira.suse.com/browse/SLE-24216

I really would like to see this in SLE 15 SP5 as default migration method, so I don't have to [document `yast2 migration` and `yast2 migration_sle`](https://susedoc.github.io/doc-sle/main/html/SLES-upgrade/cha-upgrade-online.html#sec-upgrade-online-leap-to-sle). But before this becomes supported, I think the input fields should be aligned with yast registration. (Ab)using the registration code field for the RMT server URL looks like a bad hack.

### CODE REVIEW

**Review by dgdavid — COMMENTED** [review]
It looks good. @lslezak, I guess we can simply bump version and merge it, right?

File: src/lib/migration_sle/dialogs/registration.rb [hunk 1]
```diff
@@ -115,15 +115,13 @@ def help
         text = _(<<-HELP
 <h2>Online Migration to %{to_system}</h2>
 <p>
-  This YaST module can migrate your system to the %{to_system}
-  (SLE) product. The migration is done online, in the currently
-  running system.
+  This YaST module can migrate your %{from_system} system to %{to_system}.
```
Comment (lslezak) [inline src/lib/migration_sle/dialogs/registration.rb:121]: Ah, that `from_system` placeholder is not supported, I'll adapt the code...

File: package/yast2-migration-sle.spec [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-migration-sle
-Version:        4.5.0
+Version:        4.5.2
```
Comment (dgdavid) [inline package/yast2-migration-sle.spec]: You mean 4.5.1, right?

**Review by dgdavid — CHANGES_REQUESTED** [review]

**Comment by lslezak** [comment]
I have updated the migration steps. It really first registers the OpenSUSE Leap product which is then migrated to SLES.

migration_help_text

**Review by dgdavid — APPROVED** [review]
LGTM

**Pull request merged**